src/main/java/seedu/address/commons/util/TabUtil.java

```diff
@@ -30,15 +29,9 @@ public static TabName getSwitchTabName(String tab) throws ParseException {
             tabName = TabName.PROFILE;
             break;
         default:
-            throw new ParseException(MESSAGE_INVALID_TAB);
+            throw new ParseException(MESSAGE_INVALID_ITEM_TYPE);
```

Comment:
Will this throw an inappropriate error message? Correct me if I'm mistaken, but does this throw the error message "Item type has to be either 'com', 'int', 'app' or 'me'"? If so, I believe that since we can't run `switch int`, should we revert back to the old error message / something along the lines of "Item type has to be either 'com', 'app' or 'me'"?

Reply:
hmm, I think since this is a more towards only `switch` command, I believe I should make a separate message then. Are there any other commands that have the same issue? I believe `list` and `find` as well right?

Reply:
Yup true! Sounds good!